src/app/shared/filter/date-range.component.spec.ts

```diff
@@ -1,3 +1,54 @@
-/**
- * Created by sandibarr on 9/21/17.
- */
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
💯  for setting up the stuff for tests even if there's not much to test right now